tests/api/access.test.js

```diff
@@ -22,6 +22,7 @@ const query = {
     'query searchResult($input: [SearchInput]) {\n  searchResult: search(input: $input) {\n    items\n    __typename\n  }\n}\n',
 }
 
+jest.retryTimes(3);
```

Comment:
Can this be set on `jest.config.js` ?

Reply:
I haven't seen the option to where we can configure that globally.

Reply:
- I wonder if we can set it in `globalSetup.js`
- Or maybe we could use an environment variable.

Note: the current code works, I'm just looking for ways to improve maintainability.